File: python/Gate Two/task_eight.py

sum_of_numbers = 0
continue_asking = True

while(continue_asking):
    user_input = int(input("Enter a number (Enter 0 to stop): "))
    sum_of_numbers += user_input
    if user_input == 0:
        continue_asking = False

print(f"The sum of all your numbers is {sum_of_numbers}")
    

File: java/Gate Two/TaskEight.java
import java.util.Scanner;
 
public class TaskEight{
    public static void main(String[] args){
	Scanner userInput = new Scanner(System.in);
	int sumOfNumbers = 0;
	
	boolean continueAsking = true;

	while(continueAsking){
		System.out.print("Enter a number (Enter 0 to stop): ");
		int userNumber = userInput.nextInt();
		sumOfNumbers += userNumber;
		
		if (userNumber == 0 ){
			continueAsking = false;
			}
		}
     	System.out.printf("The sum of all your numbers is %d\n", sumOfNumbers);
	}
}
